group/init/v1/schema.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I grabbed this file from https://github.com/ssbc/private-group-spec/blob/044226cdad371d30d364de319aa6a69251bb9f6b/group/init/schema.json , I hope that's accurate to ahau's usage